modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -111,42 +110,25 @@
       before { mock_facility }
 
       it 'healthcareService is populated' do
-        VCR.use_cassette('appointments/VAOS_v2/get_facility_clinics_200',
+        VCR.use_cassette('appointments/VAOS_v2/get_clinic_200',
                          match_requests_on: %i[method uri]) do
           VCR.use_cassette('appointments/VAOS_v2/get_appointment_200',
                            match_requests_on: %i[method uri]) do
             get '/mobile/v0/appointments', headers: iam_headers, params: params
           end
         end
         expect(response.body).to match_json_schema('VAOS_v2_appointments')
-        expect(response.parsed_body.dig('data', 0, 'attributes', 'healthcareService')).to eq('CHY C&P AUDIO')
+        expect(response.parsed_body.dig('data', 0, 'attributes', 'healthcareService')).to eq('MTZ-LAB (BLOOD WORK)')
       end
     end
 
-    context 'backfill clinic service no clinic data' do
+    context 'backfill clinic service uses facility id that does not exist' do
```

Comment:
new clinic service upstream service now just gives a 500 when a bad facility id is given, so I grouped the specs together.